Feed.py

```diff
@@ -243,7 +218,93 @@ def processRSS():
             time.sleep(artPause)
         time.sleep(grpPause - artPause)
 
+global artPause
+global days
+global grpPause
+global idText
+global uri
+global wait
+
+log.log('Starting Feed {0}'.format(VERSION))
+
 try:
+    arg_parser = argparse.ArgumentParser(description="Morse wire feed", parents=\
+     [\
+      config.serial_port_override, \
+      config.code_type_override, \
+      config.interface_type_override, \
+      config.sound_override, \
+      config.sounder_override, \
+      config.spacing_override, \
+      config.server_url_override, \
+      config.min_char_speed_override, \
+    # config.text_speed_override, \         # Specified as positional arg. #4
```

Comment:
There is a trade-off with this one...
With the ability to use a PyKOB recording as a source, the 'Text Speed' isn't relevant, since the code is going to be sent at the speed it was recorded. Therefore, requiring it is an extra parameter. However, it can be specified as '20' (or whatever) to satisfy the 'required' (positional) nature of it.

I'm fine with that. For the Instructograph and Dial-Up feeds that are sourced from recordings I'll just use '20' for that value.

It's probably better that it be required (and possibly ignored) than to be optional and not be what people expect.

Reply:
I think I feel the same way.  I can't imagine trying to predict what specifying it as a positional or configured value AND a command-line argument might bring.  Plus, feeds seem to have their own definite ideas of speeds already, which shouldn't be affected by the configuration.  I think having different profiles for different wires will be a more elegant way of specifying this, and THEN perhaps dropping the positional argument will make more sense.